README.md

```diff
@@ -38,6 +38,10 @@ If you wish to manually create and publish a package for Python:
 
 ## Release notes
 
+### 3.4.0
+
++ Added `WILDCARD` filter operation, which filter a field by wildcard expression.
```

Comment:
Do we have any links to explain what is the `wildcard expression`?

Reply:
Do you mean link on Wikipedia?

Reply:
I mean the link that has a definition for `wildcard expression` and that explains what you can or cannot do with wildcards. If the link to Wikipedia contains that information we can use it. But there is some spec for wildcards it is better to use a link to that spec